composer.json

```diff
@@ -39,7 +39,7 @@
     },
     "autoload-dev": {
```

Comment:
Dit kan er helemaal uit, want de normale autoloader zorgt er al voor dat de classes binnen `Tests` ook ingeladen kunnen worden.

Reply:
Klopt. Heeft wel het voordeel dat PhpStorm ziet dat het tests zijn.

Reply:
Dat ziet ie bij mij ook zonder dit?

Reply:
Is weg gehaald